File: java_project/student-management-system/demo-student/src/main/java/com/example/demo/service/StudentService.java
package com.example.demo.service;

import java.util.List;

import com.example.demo.model.Student;

public interface StudentService {

	public List<Student> listStudent();
	
	public Student selectStudent(int id);
	
	public Integer addStudent(Student student);
	
	public Integer updateStudent(Student student);
	
	public Integer removeStudent(int id);
}

File: kotlin_learning/kotlin_librarian/src/main/kotlin/com/librarian/elibrarian/service/StudentService.kt
package com.librarian.elibrarian.service

import com.librarian.elibrarian.model.Student
import java.util.*

interface StudentService {
    fun create(student: Student): Student
    fun list(): List<Student>
    fun getById(studId: Int): Optional<Student>
    fun update(student: Student): Student
    fun remove(studId: Int): Int
}